app/containers/ToDos/saga.js

```diff
@@ -15,7 +16,9 @@ export function* getToDo() {
   try {
     // Call our request helper (see 'services/axiosGetToDo')
     const toDos = yield call(getToDoList);
-    yield put(changeToDo(toDos.data));
+    const validation = schema.validate(toDos.data);
```

Comment:
Forse farei la validazione direttamente nel servizio sai?
tipo in services/getTodoList/index.js metti il servizio
in services/getTodoList/schema.js lo schema

Poi fai proxy({}).then(res => schema.validate(res)) o una roba del genere.

che dici?